tests/WE2E/test_configs/verification/config.get_obs_hpss.do_vx_det.multicyc.cycintvl_07hr_inits_vary_fcstlen_09hr.ncep-hrrr.yaml

```diff
@@ -0,0 +1,61 @@
+metadata:
+  description: |-
+    SRW App configuration file to first pull CCPA, MRMS, and NDAS observations
+    from HPSS for multiple cycles and then perform deterministic verification
+    for all cycles, including first performing vx preprocessing with METplus
+    tools such as PcpCombine and Pb2Nc.
+
+    The staged forecast data are from NCEP's operational version of the HRRR.
+
+    This test is for the scenario in which there are multiple, short (i.e.
+    shorter than 24hr), overlapping forecasts in a day.
+
+user:
+  RUN_ENVIR: community
+
+platform:
+  EXTRN_MDL_DATA_STORES: hpss
+
+workflow:
+  PREEXISTING_DIR_METHOD: rename
+  PREDEF_GRID_NAME: RRFS_CONUS_25km
+  INCR_CYCL_FREQ: 7
+  DATE_FIRST_CYCL: '2024042902'
+  DATE_LAST_CYCL: '2024043006'
+  FCST_LEN_HRS: 9
+
+rocoto:
+  tasks:
+    taskgroups: '{{ ["parm/wflow/verify_pre.yaml", "parm/wflow/verify_det.yaml"]|include }}'
+
+task_get_extrn_lbcs:
+  LBC_SPEC_INTVL_HRS: 3
+
+task_run_post:
+  POST_OUTPUT_DOMAIN_NAME: 'hrrr_ncep'
+
+verification:
+  METPLUS_VERBOSITY_LEVEL: 5
+  # Base directories in which to look for obs.  If these do not exist and/or
+  # do not contain the requied obs, create and populate them.
```

Comment:
required

```suggestion
  # do not contain the required obs, create and populate them.
```

Reply:
@MichaelLueken Thanks for catching this.  I'll fix in all the new test config files.